src/controllers/enrolment/helpers/newEnrolmentHandler.js

```diff
@@ -45,9 +46,13 @@ const newEnrolmentHandler = async (req, res) => {
     } else {
       message = await nonTrialEnrolmentHandler(req, session);
     }
-    let email = await findUserEmail(memberId);
+    let {userData,businessSessionData,businessClassData} = await findUserEmail(memberId,_id,classId);
     await session.commitTransaction();
-    SuccessfullEnrollmentEmail.send({to:email});
+    if (isTrialEnrolment) {
+      SuccessfullTrialEnrollmentEmail.send(userData,businessSessionData,businessClassData);
```

Comment:
SuccessfullTrialEnrollmentEmail.send(to, {userData,businessSessionData,businessClassData});

update this in all places with similar code.